### DISCUSSION TITLE
Feature request: Automatically apply resolution from controlnet image

### DISCUSSION THREAD

**Discussion by jordanspangenberg** [Mikubill/sd-webui-controlnet#1208] [Ideas]
I would like the option to automatically set the resolution (or adjust based on aspect ratio) when doing batch jobs. Would this be a possible toggle to add?

**Comment by syntaxerror41** [comment]
Personally I agree to automatically set aspect ratio (and display actual resolution in brackets maybe). And then be able to adjust with a scale accordingly
